src/signals/incident-management/components/TextAreaInput_b/index.test.js

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2018 - 2022 Gemeente Amsterdam
+import Enzyme, { shallow } from 'enzyme'
```

Comment:
een ideetje om dit meteen in typescript te doen?

Reply:
de test file ook? wat is het voordeel daarvan